### PR TITLE
Fixed styles

### PR DESCRIPTION
**Pull request by aalvarado**
- dashboard
- posts
- post index
- *_actions for actions class

### CODE REVIEW

File: app/views/posts/_post.html.haml [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-      %h1
-        = link_to post.title, user_post_path( post.user, post )
-      %p= authored_by post.author_username, post.created_at
-      .excerpt
-        = markdown post.excerpt
-      .post-item-actions
-        = link_to t('actions.read_more'), user_post_path( post.user, post ), :class => 'read-more'
-        \-
-        = link_to_if is_author?( post ), t('actions.edit', :item => ''), edit_dashboard_post_path( post ), :class => :edit
+
```
Comment (EnriqueVidal) [inline app/views/posts/_post.html.haml]: Please remove blank line.

File: app/views/posts/_post.html.haml [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
+= content_tag_for :div, post do
+  %h1
+    = link_to post.title, user_post_path( post.user, post )
```
Comment (EnriqueVidal) [inline app/views/posts/_post.html.haml]: This can be a single line.

**Comment by EnriqueVidal** [comment]
I'm guessing this is not yet finished, not thrilled about using the buttons nor the new colors :S I kind of like the old colors better, text on `Articles#show` is not justified and you're missing the avatar on both views. Problem with the width persists in `Articles#show`

File: app/assets/stylesheets/layout.scss [hunk 3]
```diff
@@ -219,3 +225,24 @@ footer {
...
+}
+.actions {
+  border-top: 1px solid $light_gray;
+}
+.read-more {
+  @extend .btn;
+  @extend .btn-primary;
+}
+.edit {
+  @extend .btn;
```
Comment (EnriqueVidal) [inline app/assets/stylesheets/layout.scss:301]: Not thirlled by any of these `@extend` calls we're not really adding anything else to these new styles but our compiled `coffeescript` will repeat the styles multiple times in the `css`, I see not reason not to use them separately as we were unless there is something I'm no seeing in here.

**Comment by aalvarado** [comment]
not yet finished

**Comment by aalvarado** [comment]
im still undecided on using the avatar. 

which colors are you talking about? 

articles show should not be justified. 

I'm still thinking about adding something to the right side of new_article & new_post

**Comment by EnriqueVidal** [comment]
why undecided about the avatar, h1 had a different style before you can check master for that, brownish red color and an underline. I don't like the look of the links for those pages, they standout as completely different to the rest of the app. and the buttons are huge (not even using `btn-mini`) on the other PR (the one that was closed to prevend duplicity) I had removed the text from the buttons on `Posts#show` for them to have the same width and I had used `btn-mini` to not have huge buttons.

**Comment by aalvarado** [comment]
i'll check them out.

File: app/assets/stylesheets/layout.scss (hunk 3 above)
Comment (aalvarado) [inline app/assets/stylesheets/layout.scss:301]: `btn` can be on the html. 

the `span#` cannot.

Comment (EnriqueVidal) [inline app/assets/stylesheets/layout.scss:301]: fair enough `span#` is noisy.

**Comment by EnriqueVidal** [comment]
There is a new problem, the `Page#edit`, `Page#new`, `Post#new` and `Post#edit` have now inherited the width problem `Posts#show` and `Post#index` had.

**Comment by aalvarado** [comment]
ENGLISH plz

**Pull request merged**